docs/experiments-covid.md

```diff
@@ -81,19 +101,45 @@ They are, instead:
-|:------|:-------|:---------|:---------|:---------|
-| `anserini` | `r3.fusion1` | 0.5359 | 0.8475 | 0.2293
-| `anserini` | `r3.fusion2` | 0.6100 | 0.9100 | 0.2641
-| `anserini` | `r3.rf`      | 0.6883 | 0.9750 | 0.2817
+| group | runtag | nDCG@10 | J@10 | AP   | R@1k |
+|:------|:-------|--------:|-----:|-----:|-----:|
+| `anserini` | `r3.fusion1`                       | 0.5339 | 0.8400 | 0.2283 | 0.6160
+| `anserini` | `r3.fusion1` (NIST post-processed) | 0.5359 | 0.8475 | 0.2293 | 0.6160
+| `anserini` | `r3.fusion2`                       | 0.6072 | 0.9025 | 0.2631 | 0.6441
+| `anserini` | `r3.fusion2` (NIST post-processed) | 0.6100 | 0.9100 | 0.2641 | 0.6441
+| `anserini` | `r3.rf`                            | 0.6812 | 0.9600 | 0.2787 | 0.6399
+| `anserini` | `r3.rf` (NIST post-processed)      | 0.6883 | 0.9750 | 0.2817 | 0.6399
+
+Commands for replicating above results:
+
+```
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.fusion1.txt
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.fusion2.txt
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.rf.txt
+
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.fusion1.post-processed.txt
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.fusion2.post-processed.txt
+tools/eval/trec_eval.9.0.4/trec_eval -c -M1000 -m recall.1000 -m ndcg_cut.10 -m map src/main/resources/topics-and-qrels/qrels.covid-round3.txt runs/anserini.final-r3.rf.post-processed.txt
+
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion1.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion2.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.rf.txt
+
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion1.post-processed.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.fusion2.post-processed.txt
+python tools/eval/measure_judged.py --qrels src/main/resources/topics-and-qrels/qrels.covid-round3.txt --cutoffs 10 --run runs/anserini.final-r3.rf.post-processed.txt
```

Comment:
Do we want to make this part of  `src/main/python/trec-covid/generate_round3_baselines.py` ?

Reply:
Maybe it can be done when you refactor the code

Reply:
Yes, I was planning on doing it in a subsequent round.